src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -46,44 +46,27 @@ const StyledLogo = styled.img`
   margin-left: 6px;
 `
 
-const getBrowserWalletConfig = () => {
-  const { ethereum } = window
-  const hasInjectedWallet = !!ethereum
-  const { isCoin98, isBraveWallet, isMetaMask } = ethereum || {}
-
-  if (hasInjectedWallet) {
-    if (isCoin98) {
-      const { name, iconName } = SUPPORTED_WALLETS.COIN98
-      return { name, iconName }
-    }
-
-    if (isBraveWallet) {
-      const { name, iconName } = SUPPORTED_WALLETS.BRAVE
-      return { name, iconName }
-    }
-
-    if (isMetaMask) {
-      const { name, iconName } = SUPPORTED_WALLETS.METAMASK
-      return { name, iconName }
-    }
-
-    const config = SUPPORTED_WALLETS.INJECTED
-    return {
-      name: t`your wallet`,
-      iconName: config.iconName,
-    }
-  }
-
-  return undefined
-}
+const CountdountNumber = styled.div`
+  position: absolute;
+  font-size: 30px;
+  margin-top: 25px;
+  // font-weight: 600;
+`
 
 function ConfirmationPendingContent({
   onDismiss,
   pendingText,
+  startedTime,
```

Comment:
I dont think use this is ok. This can be internal state.
```
const [count, setCount] = useState(1);
 useInterval(() => setCount(prev => prev+1), 1000);
```
When user click confirm button then we render this component, otherwise not render it

Reply:
If parent component own this state, it will re-render all child components

Reply:
state colocation

Reply:
This counter share between screen, so it must pass down to child. This component is child.